File: src/listas/basica/vPython/Node_formal.py
class Node:
    def __init__(self, value):
        self.value = value
        self.next = None

    @property
    def value(self):
        return self._value

    @value.setter
    def value(self, new_value):
        self._value = new_value

    @property
    def next(self):
        return self._next

    @next.setter
    def next(self, next_node):
        self._next = next_node

File: production/eda_23_24/listas/basica/vJavascript/Node_formal.js
class Node {
    constructor(value) {
        this.value = value;
        this.next = null;
    }

    getValue() {
        return this.value;
    }

    setValue(newValue) {
        this.value = newValue;
    }

    getNext() {
        return this.next;
    }

    setNext(next) {
        this.next = next;
    }
}

module.exports = Node;